openmdao/docs/developer_docs/index.rst

```diff
@@ -45,3 +45,20 @@ in terms of formatting and embedding code.
 
     style_guide/doc_style_guide.rst
     style_guide/sphinx_decorators.rst
+
+
+Setting Up an OpenMDAO Plugin
+-----------------------------
+
+There are several things to take into account when setting up a repository that depends on
+OpenMDAO.  Github settings, getting documentation transferred to github-pages, setting up
+continuous integration on Travis CI, and releasing software are some of the topics covered.
+
+.. toctree::
+    :maxdepth: 1
+
+    repo_guide/docs.rst
+    repo_guide/github.rst
```

Comment:
looking at the github section, it seems like maybe it should come after the travis section (since you first have to know how to build stuff on travis, and it references section of travis.yml that is introduced there)

Reply:
great feedback, thx